From: Keith Packard <keithp@keithp.com>
Date: Thu, 1 Nov 2018 13:21:05 -0700
Subject: Fix dependencies for debian build.

Removed prawn/templates, which eliminates the ability to use .pdf
files as images.

Switch concurrent-ruby dependency to ~> 1.1. concurrent-ruby is using
semantic versioning, which means any minor version above 1.1 "should"
be compatible.

Update gemspec dependencies to match debian packages

I've tested with these versions and the test suites results match the
current debian package, although there are spurious errors, e.g. 12 !=
12.0.

Signed-off-by: Keith Packard <keithp@keithp.com>
---
 asciidoctor-pdf.gemspec | 6 +++---
 lib/asciidoctor/pdf.rb  | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/asciidoctor-pdf.gemspec b/asciidoctor-pdf.gemspec
index 2afa2a1..0579aeb 100644
--- a/asciidoctor-pdf.gemspec
+++ b/asciidoctor-pdf.gemspec
@@ -33,11 +33,11 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   #s.test_files = files.grep %r/^(?:test|spec|feature)\/.*$/
 
-  s.add_runtime_dependency 'asciidoctor', '~> 2.0'
-  s.add_runtime_dependency 'prawn', '~> 2.4.0'
+  s.add_runtime_dependency 'asciidoctor', '>= 1.5.3', '< 3.0.0'
+  s.add_runtime_dependency 'prawn', '>= 2.3.0'
   # NOTE must use prawn-table from head (defined in Gemfile) for full functionality
   s.add_runtime_dependency 'prawn-table', '~> 0.2.0'
-  s.add_runtime_dependency 'prawn-templates', '~> 0.1.0'
+  # s.add_runtime_dependency 'prawn-templates', '~> 0.1.0'
   s.add_runtime_dependency 'prawn-svg', '~> 0.32.0'
   s.add_runtime_dependency 'prawn-icon', '~> 3.0.0'
   s.add_runtime_dependency 'safe_yaml', '~> 1.0.0'
diff --git a/lib/asciidoctor/pdf.rb b/lib/asciidoctor/pdf.rb
index 84f5535..475a695 100644
--- a/lib/asciidoctor/pdf.rb
+++ b/lib/asciidoctor/pdf.rb
@@ -5,7 +5,7 @@ require 'asciidoctor' unless defined? Asciidoctor.load
 require 'prawn'
 Prawn.send :remove_const, :FLOAT_PRECISION
 Prawn::FLOAT_PRECISION = 1e-3
-require 'prawn/templates'
+# require 'prawn/templates'
 begin
   require 'prawn/gmagick'
 rescue LoadError
